integration_tests/pages/additionalConditionInput.ts

```diff
@@ -86,8 +86,8 @@ export default class AdditionalConditionsInputPage extends Page {
   }
 
   nextCondition = (runAxe = true): AdditionalConditionsInputPage => {
-    cy.get(this.continueButtonId).click()
     cy.task('stubGetLicenceWithConditionToComplete', this.additionalConditionsToInput.shift())
+    cy.get(this.continueButtonId).click()
```

Comment:
From what I can tell, this was previously not actually working correctly. The 'correct' condition wasn't being loaded before the input page rendered, but because all of the input pages were built the same and we have a lot of stubbing going on, it all worked. Now that I've had to add specific routed for Curfew though, it broke, as it tried to load the normal input page before recognising that the condition in question was curfew.
By switching these calls, it now correctly recognises which condition it's currently looking at